BugReporter/GitHubUrlBuilder.cs

```diff
@@ -22,7 +22,7 @@ public GitHubUrlBuilder(IErrorReportMarkDownBodyBuilder errorReportMarkDownBodyB
         /// Generates a URL to create a new issue on GitHub.
         /// </summary>
         /// <see href="https://help.github.com/en/articles/about-automation-for-issues-and-pull-requests-with-query-parameters"/>
-        public string? Build(string url, SerializableException exception, string? environmentInfo, string? additionalInfo)
+        public string? Build(string url, SerializableException exception, string exceptionInfo, string? environmentInfo, string? additionalInfo)
```

Comment:
What's the difference between `exception` and `exceptionInfo` and why the latter can't be derived from the former?

Reply:
GE specific exceptions like `GitExtUtils.ExternalOperationException` are not available in BugReporter, so the summary for GE internal exceptions are passed as a string.

Reply:
Please add this comment as xml-doc

Reply:
done